src/Views/batchprocessor.html

```diff
@@ -104,16 +136,53 @@ <h3>Select Basin Characteristics</h3>
                         </tr>
                     </tbody>
                 </table>
+            </div><!-- End parameter selection -->
 
-            </div>
-            <!-- End submit batch -->
+            <!-- Start user information -->
+            <div style="padding-top: 15px;" class="row">
+                <div class="col-md-5"> ID Field: 
+                    <i ng-mouseover="showTooltip = true" ng-mouseleave="showTooltip = false" class="fa fa-question-circle"></i>
+                    <span ng-show="showTooltip" class="report-tooltip">
+                        <span>This is the field heading from the shapefile of points that is unique to each point and allows the
+                            data to be
+                            joined with the batch results.</span>
+                    </span>
+                    <input type="text" id="idField" ng-model="vm.submitBatchData.idField" required>
+                </div>
+                <div class="col-md-7">
+                    <span> Email Address: 
+                        <input type="text" id="emailField" ng-model="vm.submitBatchData.email" required></span>
+                </div>
+            </div> <!-- End user information-->
 
-            <!-- Start batch status -->
-            <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
-                <p>This is the batch status tab</p>
+            <!-- Create a file upload button and specify it needs to be a zipped shapefile -->
+            <div style="padding-top: 15px;">
+                <span>Upload Shapefile (.zip):</span><br />
+                <label for="shapefile" style="border: 1px solid #ccc; display: inline-block; padding: 6px 12px; cursor: pointer;"> Select file</label>

```

Comment:
Thoughts on setting the style to `2px solid #FBEC5D` if no file is uploaded?
![image](https://user-images.githubusercontent.com/40237491/231619327-4320eba2-2740-4cc6-9066-11a0322440d6.png)